### PR TITLE
add conda recipe

### PR DESCRIPTION
**Pull request by Kvieta1990**
Add conda recipe for building conda package. This is for further implementing pystog into addie. After checking, fourier filter of pystog is not available in mantid so we have to go this way, i.e. we build a conda package which we will add to dependency list of addie.

PS: I uploaded this conda package to my own anaconda channel - `apw247`.

### CODE REVIEW

File: conda.recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package:
+  name: pystog
+  version: 1.0.0
```
Comment (marshallmcdonnell) [inline conda.recipe/meta.yaml:3]: PyStoG is on version 0.2.6 (see [PyPi release history](https://pypi.org/project/pystog/#history) for PyStoG)

Shouldn't have to hard-code the version, we should pick it up from tags in the repository

File: conda.recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+build:
+  number: 1
+
+source:
+  path: ..
+
+requirements:
+  build:
```
Comment (marshallmcdonnell) [inline conda.recipe/meta.yaml:12]: We do have more build requirements. One that comes to mind is testing dependencies and linting.

File: conda.recipe/build.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+python setup.py install
```
Comment (marshallmcdonnell) [inline conda.recipe/build.sh:1]: I think we can probably get everything defined in just the `meta.yaml`, no need for separate build script.

**Review by marshallmcdonnell — CHANGES_REQUESTED** [review]
Thanks for starting this work!
I do think we need to fix the recipe, though.
See my comments for issues we need to address.

Also, I did view [your upload of PyStoG on your Anaconda channel](https://anaconda.org/apw247/pystog). I think we should switch to a `noarch` vs. providing all 9 OS specific builds.

The way to specify `noarch` can be found [here](https://github.com/neutrons/mantid_total_scattering/blob/master/conda.recipe/mantid-total-scattering-python-wrapper/meta.yaml#L11)

Thanks again and glad to work on you with this!

File: conda.recipe/meta.yaml (hunk 2 above)
Comment (marshallmcdonnell) [reply conda.recipe/meta.yaml:12]: Actually, I take this comment back, nevermind. Sorry.

File: conda.recipe/meta.yaml (hunk 1 above)
Comment (marshallmcdonnell) [reply conda.recipe/meta.yaml:3]: The chunk of code found [here](https://github.com/neutrons/mantid_total_scattering/blob/master/conda.recipe/mantid-total-scattering-python-wrapper/meta.yaml#L1-L5) accomplishes that.

**Comment by marshallmcdonnell** [comment]
Just adding a note that we will want to:
- [ ]  delete the pystog package located [on your channel](https://anaconda.org/apw247/pystog) for "clean up".

No rush, just documenting it in this issue.

**Comment by marshallmcdonnell** [comment]
Okay, think this is ready.

I'm going to address issues #36 in a separate branch and add the CD for anaconda to that one since we don't want to implement via travis CI only to switch to GitHub Actions.

**Review by marshallmcdonnell — APPROVED** [review]

File: conda.recipe/meta.yaml (hunk 1 above)
Comment (marshallmcdonnell) [reply conda.recipe/meta.yaml]: Done

File: conda.recipe/build.sh (hunk 3 above)
Comment (marshallmcdonnell) [reply conda.recipe/build.sh]: Done

**Comment by Kvieta1990** [comment]
Thank you Marshall for working on this! So what I need to do is to delete the `pystog` package from my anaconda channel, right? Anything else I need to do?

**Comment by marshallmcdonnell** [comment]
Nope that should do it for now!

**Pull request merged**